.gitignore

```diff
@@ -6,4 +6,5 @@ dockerfiles/run/**/keystore
 dockerfiles/run/**/glowroot
 .m2
 test-run.log
-build
\ No newline at end of file
+build
+.mvn/.gradle-enterprise/gradle-enterprise-workspace-id
```

Comment:
I would recommend ignoring all of `.mvn/.gradle-enterprise`. While the only file currently written is the one you are explicitly ignoring, it is possible that later versions of the extension may use this directory.

Reply:
done